library/src/main/java/com/northconcepts/templatemaster/content/Util.java

```diff
@@ -135,5 +140,49 @@ public static int getCharacterCount(String string, char searchChar) {
         
         return count;
     }
-    
+
+    public static String emptyToNull(String value) {
+        if (isEmpty(value)) {
+            return null;
+        }
+        return value;
+    }
+
+    public static String nullToEmpty(String value) {
+        if (value == null) {
+            return "";
+        }
+        return value;
+    }
+
+    public static boolean equals(Object o1, Object o2) {
+        return Util.equals(o1, o2);
```

Comment:
@gvjoshi25 is this a recursive call?  Is `Util` this class?

Reply:
Yes, it is recursive call since a long time. :(
Anyway, I am planning to use `java.util.Objects`

```
1. Objects.equals(o1, o2);
2. Objects.deepEquals(o1, o2);
```

I suggest, we should go with deepEquals. Or you have any other library in mind?

Actually, we don't even need these methods in our TM Util as they are provided by java.

Reply:
You're welcome to remove them.  Thanks.

Reply:
Removed